src/include/wt_internal.h

```diff
@@ -369,14 +368,16 @@ union __wt_rand_state;
 #include "session.h"			/* required by connection.h */
 #include "connection.h"
 
-#include "extern.h"
+#include "extern.h"			/* required by error.h */
 #ifdef _WIN32
 #include "extern_win.h"
 #else
 #include "extern_posix.h"
 #endif
 #include "verify_build.h"
 
+#include "error.h"
+
```

Comment:
@dgottlieb, is this change still necessary?

Reply:
`error.h` now holds the `__wt_err[x]` macros which call into `__wt_err[x]_func`. Those methods are declared in `extern.h`, so we pulled `error.h` below it.

Reply:
Yes, but those are #defines, so it doesn't matter?

Reply:
You're right, I moved it back and it works.

Which leaves me wondering what the compile problem was in the first place!